tests/pylint.py

```diff
@@ -0,0 +1,3 @@
+from cv2 import getStructuringElement, MORPH_ELLIPSE
+
+a = getStructuringElement(MORPH_ELLIPSE, (3, 3))
```

Comment:
Why do you need it? I propose to use one of OpenCV samples, if you need entry point.

Reply:
Tests in `opencv/modules/python/test/` doesn't pass a pylint test, there are some issues.

Reply:
https://pullrequest.opencv.org/buildbot/builders/precommit_docs/builds/34604/steps/Run%20check_pylint/logs/stdio

Reply:
Currently, we cannot do the same logic using `cmake`, because the test part doesn't have the `build` folder in opencv.

Added `pylintrc` in this repository from `opencv/platforms/scripts/pylintrc` (didn't provide it via flag, because the current content won't show any output for our test).

Btw, changed the test file from a custom script to an opencv sample (`opencv/samples/python/squares.py`).